src/modules/auth/auth.controller.ts

```diff
@@ -13,5 +14,64 @@ export class AuthController {
     return this.authService.register(body);
   }
 
+  @Post('login')
+  async login(@Body() body: LoginDto, @Res() res: ExpressResponse) {
+    try {
+      const { accessToken, refreshToken } = await this.authService.login(body);
+      this.setCookies(res, accessToken, refreshToken);
+      res.status(HttpStatus.OK).end('đăng nhập thành công');
+    } catch (error) {
+      res.status(HttpStatus.UNAUTHORIZED).end('đăng nhập thất bại');
+    }
+  }
+
+  @Post('logout')
+  async logout(@Res() res: ExpressResponse, @Req() req: ExpressRequest) {
+    try {
+      const accessToken = req.cookies['access_token'];
```

Comment:
The term access_token is being used in several places. Should move it into a constant.